cmd/ltctl/main.go

```diff
@@ -204,6 +204,38 @@ func main() {
 	}
 	rootCmd.AddCommand(collectCmd)
 
+	reportCmd := &cobra.Command{
+		Use:   "report",
+		Short: "Get or compare reports from load tests",
+	}
+
+	genReport := &cobra.Command{
+		Use:     "generate",
+		Short:   "Generate a report from a load test",
+		Example: "ltctl report generate --start-time=1591858735 --end-time=1591858960 --output=base.out --label=base",
+		RunE:    RunGenerateReportCmdF,
+	}
+	genReport.Flags().Int64P("start-time", "s", 0, "Start time in epoch miliseconds.")
+	genReport.Flags().Int64P("end-time", "e", 0, "End time in epoch miliseconds.")
```

Comment:
Yeah, I had  thought about that. The only thing is the part about timezones and the timestamp format. Passing epoch timestamps is a simple format and it will always be UTC. Passing friendly timestamps will require them to be passed in a specific format and then needs to be converted to UTC.

One could argue that we already need to convert to epoch timestamp, so there's no difference, and I agree.

Getting the timestamp from Grafana is really simple. One just needs to click on the graph and grab the timestamp. Just that needs to be converted to UTC.

I think given that there's no extra benefit, I am leaning towards formatted timestamps. Will change. Thanks for the suggestion.